kraken-app/kraken-app-portal/package.json

```diff
@@ -1,30 +1,38 @@
 {
-  "name": "kraken-app-portal",
+  "name": "kraken-portal",
   "private": true,
-  "version": "1.0.2",
+  "version": "0.230.0",
```

Comment:
@DaveXiong TBD which version number to include.

Reply:
`1.0.2` will be used at the moment, pls take note `kraken` will use the same version for FE and BE.
And `version` will be increased manually by the admin.

Reply:
@DaveXiong done